### ISSUE TITLE
ReCogDrive-VLM-DriveLM

### ISSUE THREAD

**Issue by krishnakanthnakka** [xiaomi-research/recogdrive#43]
Hello Authors

Can you please tell if this model is https://huggingface.co/owl10/ReCogDrive-VLM-DriveLM is specially trained on only DriveLM dataset? And what is the difference b/w this model and https://huggingface.co/owl10/ReCogDrive-VLM-8B/tree/main ? Can you please clarify about their training datasets?

Thanks

**Comment by krishnakanthnakka** [comment]
Hello authors,

additionally, can you please tell if the prompt for quality control is present in the codebase?

`Quality Control. Finally, we implement a rigorous filtering step. Qwen2.5-VL assigns each QA pair a quality score based on predefined linguistic and semantic criteria. Any pair scoring below 60 is discarded`

**Comment by owl-10** [comment]
> Hello Authors
> 
> Can you please tell if this model is https://huggingface.co/owl10/ReCogDrive-VLM-DriveLM is specially trained on only DriveLM dataset? And what is the difference b/w this model and https://huggingface.co/owl10/ReCogDrive-VLM-8B/tree/main ? Can you please clarify about their training datasets?
> 
> Thanks

Hello,

Regarding the difference between the two models, we initially followed the RoboTron-Drive convention by converting the center-point box coordinates in DriveLM into boxes defined by the top-left and bottom-right corners. We then reformatted them into our predefined structure, for example:

`<BACK_RIGHT VIEW>[481, 489, 655, 717]`

You can find this processed DriveLM format here:  
https://huggingface.co/datasets/owl10/ReCogDrive_Pretraining/blob/main/DriveLM/drivelm_change_box_type.jsonl

All datasets we processed used this converted format, including those used to train the ReCogDrive-Navsim evaluation model. The model trained with this convention is:  
https://huggingface.co/owl10/ReCogDrive-VLM-8B/tree/main

However, we later discovered that the DriveLM test set does not provide box coordinates in the converted (top-left / bottom-right) format. To properly evaluate on DriveLM, we retrained a version of the model using the original DriveLM box format, for example:

`<c3,CAM_FRONT,1180.8,541.7>`

You can find the original-format DriveLM data here:  
https://huggingface.co/datasets/owl10/ReCogDrive_Pretraining/blob/main/DriveLM/drivelm_original_box.jsonl

This retrained model is:  
https://huggingface.co/owl10/ReCogDrive-VLM-DriveLM

This is the checkpoint intended to reproduce our reported DriveLM evaluation results. If you do not need DriveLM evaluation specifically, the practical difference between the two models is relatively small.

Thanks!
